backend/fleet_management/tests/test_drives.py

```diff
@@ -0,0 +1,96 @@
+from datetime import date
+
+from django.contrib.auth.models import Group
+from django.test import override_settings
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from fleet_management.constants import Groups
+from fleet_management.factories import (
+    DriveFactory,
+    PassengerFactory,
+    CarFactory,
+    ProjectFactory,
+    UserFactory,
+)
+from fleet_management.models import Drive
+
+
+class DrivesApiTest(APITestCase):
```

Comment:
You abandoned `test_get_all_drives`. Is that on purpose?

Reply:
@magul 
Yes, get_queryset in DriveView filter Drives in response by user_id. 
From what I understand if there is more than one driver and these drivers have any drives then it's not possible to get all drives by them.

So I conclude only test_can_retrieve_only_my_drives is testing what actually is happening in DriveView.